requirements.txt

```diff
@@ -8,7 +8,7 @@ slackeventsapi==3.0.1
 slackclient==2.9.4
 flask-cors
 asgiref==3.5.2
-lightning==1.8.*
```

Comment:
what should we pin here @Borda ?

Reply:
after release, I would set it as `1.9.*`

Reply:
@Borda shall we merge this since 1.9 has been released